### DISCUSSION TITLE
액셀 자동화 프로그램 질문입니다

### DISCUSSION THREAD

**Discussion by sayaking123** [community/community#59406] [Programming Help]
### Body

chat gpt를 이용해 액셀 자동화 프로그램을 만들고있는데 
자동화를 할려는 작업은 바코드 스캐너로 바코드 스캔시 액셀에 바코드가 입력되고나서 
 다른액셀파일에 바코드 값 행에 있는 특정 정보를 가져와서
현재 액셀파일에 자동입력되게 할려고합니다 
예를 하나 들어보자면 바코드 스캔 > 바코드값이 B2에 입력됨> 입력된 바코드 값을 자동으로 A라는 다른액셀파일에서 해당 바코드가
적혀있는 행으로 이동후 특정 정보를가져와서 바코드값이 입력되있는 B2행의 다른열에 정보를 자동으로 입력하는걸 만들고싶습니다.
GPT에게 물어보았지만 제가 원하는 그림이 나오지 않고있습니다 도와주세용..
코딩에 지식이 깊지 않아 
gpt에 의존하고있는데 구현이 되지 않고있습니다 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by queenofcorgis** [comment]
Hi @sayaking123 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
